### DISCUSSION TITLE
[Feature Request] EthOS

### DISCUSSION THREAD

**Discussion by Teklock** [iNavFlight/OpenTX-Telemetry-Widget#93] [Ideas]
Any chance this LUA script can be made to run on FrSky's new line of radios running EthOS?

**Comment by stronnag** [comment]
Unless 
* EthOS is API compatible with OTX/ETX; and
* FrSky wishes to supply the developers with sample hardware and API documentation

then there is little chance.

**Comment by Teklock** [comment]
From what I understand, FrSky did supply hardware for the devs over at https://github.com/yaapu/FrskyTelemetryScript to port Yaapu to EthOS.

If you can, checkout https://www.rcgroups.com/forums/showthread.php?4018791-FrSky-ETHOS-Lua-Script-Programming and see how difficult it would be for you guys to make your wonderful script work with Ethos.  Thanks for the reply!

**Comment by craigywsm** [comment]
I would be more than happy to donate if developers could get this to work

**Comment by miggy1952** [comment]
I would also like this Lua for Ethos

**Comment by jasc76** [comment]
+1

**Comment by craigywsm** [comment]
Hopefully with improvements to lua support in 1.5 this might be possible.

**Reply by craigywsm** [reply]
The simulator also support the LUA scripts

**Comment by EmanuelStassar** [comment]
As I recently upgraded to an Ethos radio, I would really like this. I can’t program but am more than willing to help with testing.

**Comment by WhiskeyGolf** [comment]
Same here! I would also be very happy about the widget on Ethos. It is the only thing I am missing in the transition to Ethos. I would also like to test it. I have an x20pro. Thank you

**Comment by pedrocgr** [comment]
Support for EthOS would be awesome!
